CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+* Created benchmarks for the search functionality ([#2740])
+
```

Comment:
It's not a user-facing change, so doesn't actually warrant a CHANGELOG. I also added the `Skip Changelog` label to suppress the CI check.

```suggestion
```